packages/typescript/src/components/ClassDeclaration.tsx

```diff
@@ -137,14 +153,27 @@ export interface ClassMethodProps extends ClassMemberProps {
 
 export function ClassMethod(props: ClassMethodProps) {
   const returnType = props.returnType && <>: {props.returnType}</>;
+  const [docProps, rest] = splitProps(props, ["doc"]);
```

Comment:
This splitProps seems unnecessary.

Reply:
I need to take out docs from props otherwise the memeber will try to render them again. Member also renders docs because a non method member needs simple JSDoc